src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -1,149 +1,154 @@
 package frc.robot.subsystems.arm;
 
```

Comment:
ngl I don't have the badnwidth to look at this rn. to confirm, you just ran the formatter and didn't actually make any changes, right?